europace_security.yaml

```
@@ -14,99 +14,107 @@ flows:
-        ## Baufinanzierungsvorgänge schreiben
+        ## BaufiSmart-Vorgänge anlegen
       baufinanzierung:angebot:ermitteln: |
-        ## Baufinanzierungsangebote ermitteln
+        ## BaufiSmart-Angebote ermitteln
       baufinanzierung:angebot:loeschen: |
-        ## Baufinanzierungsangebote loeschen
+        ## gespeicherte BaufiSmart-Angebote löschen
       baufinanzierung:ereignis:lesen: |
-        ## Baufinanzierungsereignisse lesen
+        ## BaufiSmart-Ereignisse lesen
       baufinanzierung:antrag:lesen: |
-        ## Baufinanzierungsanträge lesen
+        ## BaufiSmart-Anträge lesen
       baufinanzierung:antrag:schreiben: |
-        ## Baufinanzierungsanträge schreiben
+        ## BaufiSmart-Anträge verändern
+        Der Antragsstatus und der Sachbearbeiter können verändert sowie die Produktanbieter-Renferenz ergänzt werden.
+      
       baufinanzierung:produktkonfiguration:lesen: |
         ## Produktkonfiguration lesen
+        für Product-Cockpit
       baufinanzierung:produktkonfiguration:schreiben: |
-        ## Produktkonfiguration schreiben
+        ## Produktkonfiguration verändern
+        für Product-Cockpit
       baufinanzierung:produktkonfiguration:freigeben: |
         ## Produktkonfiguration freigeben
+        für Product-Cockpit
       baufinanzierung:produkt:lesen: |
         ## Produkt lesen
+        für Product-Cockpit
       baufinanzierung:produkt:schreiben: |
-        ## Produkt schreiben
+        ## Produkt verändern
+        für Product-Cockpit
       baufinanzierung:produktanbieter:lesen: |
         ## Produktanbieter lesen
+        für Product-Cockpit
       baufinanzierung:produktanbieter:schreiben: |
-        ## Produktanbieter schreiben
+        ## Produktanbieter verändern
+        für Product-Cockpit
 
       privatkredit:angebot:ermitteln: |
-        ## Kreditsmartangebote ermitteln
-        Der Client kann Ratenkredit-Angebote und Schaufensterkonditionen ermitteln.
+        ## KreditSmart-Angebote ermitteln
+        Ratenkredit-Angebote und -Schaufensterkonditionen können ermittelt werden.
       privatkredit:antrag:schreiben: |
-        ## Kreditsmartanträge schreiben
-        Der Client kann den Antragsstatus verändern oder den aktuellen Status um Zusatzinformationen ergänzen.
+        ## KreditSmart-Anträge verändern
+        Der Antragsstatus kann verändert oder der aktuelle Status um Zusatzinformationen ergänzt werden.
       privatkredit:vorgang:lesen: |
-        ## Kreditsmartvorgänge lesen
-        Der Client kann KreditSmart-Vorgänge automatisiert auslesen.
+        ## KreditSmart-Vorgänge lesen
       privatkredit:vorgang:schreiben: |
-        ## Kreditsmartvorgänge schreiben
-        Der Client kann KreditSmart-Vorgänge automatisiert anlegen.
+        ## KreditSmart-Vorgänge anlegen
 
       unterlagen:dokument:lesen: |
-        ## Dokumente lesen
-        Der Client kann hochgeladene Dokumente eines Vorgangs abrufen.
+        ## Unterlagen-Dokumente herunterladen
```

```suggestion
        ## Unterlagendokumente herunterladen
```
Ist kein existierendes Wort, sollte meiner Meinung nach aber trotzdem zusammengeschreiben werden.